src/main/java/hudson/plugins/view/dashboard/stats/StatJobs.java

```diff
@@ -64,11 +64,8 @@ public static HealthStatus getHealthStatus(Job job) {
          if (score < 80) {
             return HEALTH_60_TO_79;
          }
-         if (score >= 80) {
-            return nBuilds != 0 ? HEALTH_OVER_80 : HEALTH_UNKNOWN;
-         }
 
-         return HEALTH_UNKNOWN;
```

Comment:
the old line 71 would never be called as `score` will always be either `<80` or `>= 80` so the `return` statement will always have been called.